### PR TITLE
sound_cloud.js: remove https from the image.

### PR DESCRIPTION
**Pull request by jagtalon**
CC @bsstoner

### CODE REVIEW

**Comment by jagtalon** [comment]
Hot fix for this:

screen shot 2015-02-16 at 1 54 01 pm

Changing HTTPS to HTTP fixes it

**Comment by bsstoner** [comment]
:+1:

File: share/spice/sound_cloud/sound_cloud.js [hunk 1]
```diff
@@ -51,6 +51,8 @@
                     // Get the larger image for our IA.
                     image = image.replace(/large\.jpg/, "t200x200.jpg");
                 }
+                
+                image = image.replace(/^https/, "http");
```
Comment (moollaza) [inline share/spice/sound_cloud/sound_cloud.js]: might as well use `DDG.toHTTP()`?

Comment (jagtalon) [inline share/spice/sound_cloud/sound_cloud.js]: @moollaza ah didn't know this existed. Thanks!

**Pull request merged**